### PR TITLE
Change spec to use criminal's incomes as additional rewards

### PR DESCRIPTION
**Pull request by sgkim126**
Hi,
Currently, the informant receives deposit and rewards of the criminal as a prize.
However, it incentives for informants to wait for reporting to maximize their profits.

So I propose giving the deposit to the informant and using the incomes as additional rewards for [diligent validators](https://github.com/CodeChain-io/codechain/blob/master/spec/Dynamic-Validator.md#Validator-Reward).

### CODE REVIEW

**Comment by sgkim126** [comment]
FYI, `intermediate_rewards` is cleared at the beginning of the term, not the end of the term, because calculating the penalty needs the signature of the block.

**Comment by kseo** [comment]
LGTM. Let's wait for @ScarletBlue's review.

File: spec/Dynamic-Validator.md [hunk 1]
```diff
@@ -206,7 +206,8 @@ The transaction that reports a double vote have occurred before **WITHDRAW_DELAY
 
 The criminal loses all his deposit and rewards and is banned immediately; it is the only case where a validator set is changed during the term.
 
-The informant receives all deposit and rewards(TBD) as prize money immediately.
+The informant receives the deposit of the criminal as prize money immediately.
+The express fee which the criminal earns would be used as additional rewards for diligent validators.
```
Comment (ScarletBlue) [inline spec/Dynamic-Validator.md]: The express fee that the criminal would earn is used as additional rewards for diligent validators.

File: spec/Dynamic-Validator.md [hunk 2]
```diff
@@ -206,7 +206,8 @@ The transaction that reports a double vote have occurred before **WITHDRAW_DELAY
 
 The criminal loses all his deposit and rewards and is banned immediately; it is the only case where a validator set is changed during the term.
 
-The informant receives all deposit and rewards(TBD) as prize money immediately.
+The informant receives the deposit of the criminal as prize money immediately.
+The express fee that the criminal would earn is used as additional rewards for diligent validators.
```
Comment (ScarletBlue) [inline spec/Dynamic-Validator.md]: The express fee that the criminal would have earned is used as additional rewards for diligent validators.

**Pull request merged**